modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/in_progress_forms_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module V0
+    class InProgressFormsController < ApplicationController
+      def update
+        form = InProgressForm.where(form_id:, user_uuid: @current_user.uuid).first_or_initialize
+        form.update!(form_data: params[:form_data] || params[:formData], metadata: params[:metadata])
```

Comment:
Why can the data be either `form_data` or `formData`? Why not just one param?

Reply:
Let's see if we can answer this without too much effort. The first [git blame](https://github.com/department-of-veterans-affairs/vets-api/commit/81dbd6df2814729c64a303eb911fb1901d06581c) shows this to get started.